config/settings.yml

```diff
@@ -1571,33 +1560,17 @@ check_in:
   authentication:
     max_auth_retry_limit: 3
     retry_attempt_expiry: 604800 # 7 Days
-  chip_api:
```

Comment:
👍 looks like `chip_api_v2` is what's used now

Reply:
yes, that's correct. the changes LGTM for check_in settings.